### PR TITLE
[Fix][Flink]Fix local model bug and FlinkSql task type selected by default

### PR DESCRIPTION
**Pull request by leechor**

### CODE REVIEW

File: dinky-executor/src/main/java/org/dinky/executor/ExecutorConfig.java [hunk 1]
```diff
@@ -295,7 +260,7 @@ public String getJobManagerAddress() {
     }
 
     public boolean isRemote() {
-        return Asserts.isNotNullString(this.getHost());
+        return !GatewayType.LOCAL.equalsValue(type);
     }
```
Comment (aiwenmo) [inline dinky-executor/src/main/java/org/dinky/executor/ExecutorConfig.java]: Remote: standalone, yarn-session,k8s-session;
Local:local,yarn-perjob,yarn-application,k8s-application

File: dinky-executor/src/main/java/org/dinky/executor/ExecutorFactory.java [hunk 2]
```diff
@@ -27,7 +27,9 @@
  **/
 public final class ExecutorFactory {
 
-    public static Executor getExecutor() {
+    private ExecutorFactory() {}
+
+    public static Executor getLocalStreamExecutor() {
         return new LocalStreamExecutor(ExecutorConfig.DEFAULT);
```
Comment (aiwenmo) [inline dinky-executor/src/main/java/org/dinky/executor/ExecutorFactory.java]: The default result of getExecutor is LocalStreamExecutor, No need for explicit explanation again.

**Review by aiwenmo — CHANGES_REQUESTED** [review]

**Review by Zzm0809 — APPROVED** [review]
LGTM

**Review by aiwenmo — APPROVED** [review]
LGTM

**Pull request merged**